Assets/MixedRealityToolkit-SDK/StandardAssets/Materials/MRTK_Standard_Emerald_Transparent.mat

```diff
@@ -0,0 +1,144 @@
+%YAML 1.1
```

Comment:
should we add transparent versions of all of the std materials?

Reply:
Should we?  I just did this one for the teleporter cursor

Reply:
Probably a task we can log for later.

Reply:
> Should we? I just did this one for the teleporter cursor

Perhaps a rename is more reasonable: MRTK_Default_Teleport_Cursor.mat

Reply:
I just wanted to use a standard asset so we wouldn't make too many one off materials like this.